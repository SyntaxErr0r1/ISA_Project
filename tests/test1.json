{
    "args": "-f tests/test1.feed -T -a -u",
    "feedFile": [
        "what-if.xkcd.com.xml"
    ],
    "feedUrl": [
        "http://what-if.xkcd.com/feed.atom"
    ],
    "returnCode": 0
}