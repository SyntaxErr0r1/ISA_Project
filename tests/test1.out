*** what if? ***
Transatlantic Car Rental
Autor:  xkcd
Autor (email): whatif@xkcd.com
URL: https://what-if.xkcd.com/160/
Aktualizace: 2022-09-06T00:00:00Z

Hailstones
Autor:  xkcd
Autor (email): whatif@xkcd.com
URL: https://what-if.xkcd.com/159/
Aktualizace: 2022-07-06T00:00:00Z

Hot Banana
Autor:  xkcd
Autor (email): whatif@xkcd.com
URL: https://what-if.xkcd.com/158/
Aktualizace: 2022-05-04T00:00:00Z

Earth-Moon Fire Pole
Autor:  xkcd
Autor (email): whatif@xkcd.com
URL: https://what-if.xkcd.com/157/
Aktualizace: 2018-05-21T00:00:00Z

Electrofishing for Whales
Autor:  xkcd
Autor (email): whatif@xkcd.com
URL: https://what-if.xkcd.com/156/
Aktualizace: 2017-03-09T00:00:00Z